### DISCUSSION TITLE
[Feature Request] Allow filters to target specific users.

### DISCUSSION THREAD

**Discussion by thirashima** [mastodon/mastodon#22528] [General]
It would be great to be able to target a filter rule to a specific user. So, for example, if that one guy you follow is live tooting a sports event that you don't care about, you could mute "touchdown from: sportsguy" for the next few hours, without affecting the rest of your timeline (this is obviously an imperfect example, but you probably get the idea).

**Comment by eadio** [comment]
你的邮件我已收到，我会尽快处理。。。。
